src/main/java/com/shippingcostestimator/enterprise/PackageEstimatorController.java

```diff
@@ -55,12 +55,11 @@ public String saveEstimate(Shipment shipment){
 
     /***
      * Handle the shipment ("/shipment") endpoint and return all Shipment objects that have been created from data specified by the user.
-     * @return
+     * @return all shipments in the list.
      */
     @GetMapping("/shipment")
     @ResponseBody
     public List<Shipment> fetchAllShipments(){
-
         return shipmentService.fetchAllShipments();
 
```

Comment:
Fixed format for method body for the fetchAllShipments method.  Makes it easier to read. [Ref](https://medium.com/@rhamedy/a-short-summary-of-java-coding-best-practices-31283d0167d3)